Options_Defaults.lua

```diff
@@ -114,6 +114,10 @@ local TIP_LEFT = "TIP_LEFT"
 local TIP_RIGHT = "TIP_RIGHT"
 local TIP_HIDDEN = "TIP_HIDDEN"
 
+-- Tooltip activation
+local TIP_ACTIVATION_HOVER = "TIP_ACTIVATION_HOVER"
+local TIP_ACTIVATION_CLICK = "TIP_ACTIVATION_CLICK"
+
```

Comment:
Already defined in SharedConstants, so we should always refer to that instead of re-defining it :)

Reply:
I agree with this, but as with my comment on the previous snippet, this also has some previous duplication that might be candidates for refactoring.

Reply:
True and those will be eradicated as soon as I have more time. If it's not too much of a bother, could you please change it regardless? Thanks :)